internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -111,7 +111,7 @@ func NewApplicationSnapshotImage(ctx context.Context, url string, p policy.Polic
 }
 
 // fetchPolicySources returns an array of policy sources
-func fetchPolicySources(s *ecc.Source) ([]source.PolicySource, error) {
+func fetchPolicySources(s ecc.Source) ([]source.PolicySource, error) {
```

Comment:
What's the reason here? Don't pass by reference if not needed?

Reply:
Yeah, the method doesn't mutate, it was used in the loop (see below)